src/mocks/botframework/index.js

```diff
@@ -44,6 +44,9 @@ if (!webhookurl) {
   }
 }
 
+const botHealthCheckVerb = process.env.BOTIUM_BOTFRAMEWORK_HEALTH_CHECK_VERB || 'POST'
+const botHealthCheckUrl = process.env.BOTIUM_BOTFRAMEWORK_HEALTH_CHECK_URL || webhookurl
+
```

Comment:
Please add option to only use the url path for the health check - if the HEALTH_CHECK_URL is not a full url, then prefix with the webhook-hostname and webhook-port - because full urls will most likely not work, if the hostname is something different than botium (this hostname is assigned automatically by botium docker environment)